packages/manager/src/features/Linodes/LinodesCreate/LinodeCreateContainer.tsx

```diff
@@ -995,11 +998,13 @@ const actionsAndLabels = {
   fromStackScript: { action: 'stackscript', labelPayloadKey: 'stackscript_id' },
 };
 
-const handleAnalytics = (
-  type: CreateTypes,
-  payload: CreateLinodeRequest,
-  label?: string
-) => {
+const handleAnalytics = (details: {
+  fromLinode?: Linode;
```

Comment:
```suggestion
  linode?: Linode;
```
`fromLinode` feels misleading because I would expect a boolean, and we're receiving an actual linode.

Reply:
How about `clonedLinode`? -- I feel that's even more explicit.

Reply:
A little misleading, though, since this function is still going to run for other methods of creating a linode, and some of those do have linodes associated with them. (Create from Backup, for example, and the event will fire and include the powered on/off data.)